Assets/Scripts/Buddy/BehaviorPhysicsController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[RequireComponent( typeof( ActorPhysics ) )]
+public class BehaviorPhysicsController : MonoBehaviour
+{
```

Comment:
I figured this would be something that's just solved by level design: Just don't have places in the level for the buddies to fall. But, if we eventually want to make this more robust, it's pretty easy to do so.